doc/examples/segmentation/plot_mask_slic.py

```diff
@@ -1,24 +1,29 @@
 """
 ======================
-maskSLIC Demonstration
+Apply maskSLIC vs SLIC
 ======================
 
 This example is about comparing the segmentations obtained using the
 plain SLIC method [1]_ and its masked version maskSLIC [2]_.
 
+To illustrate these segmentation methods, we use an image of biological tissue
+with immunohistochemical (IHC) staining. The same biomedical image is used in
+the example on how to
+:ref:`sphx_glr_auto_examples_color_exposure_plot_ihc_color_separation.py`.
```

Comment:
I had to check the docs to see how this got rendered and notice "Separate" is capitalized, but I don't think it is a big issue if that is not easy to fix. 

Does it work if you use the following form?:

```
:ref:`sphx_glr_auto_examples_color_exposure_plot_ihc_color_separation.py <separate colors in immunohistochemical staining>`.
```

That would also be robust to future changes to the title in the other file. 

link to the rendered doc:
https://1426-2014929-gh.circle-artifacts.com/0/doc/build/html/auto_examples/segmentation/plot_mask_slic.html#sphx-glr-auto-examples-segmentation-plot-mask-slic-py

Reply:
Oh, yes. I expected this, for I used the same syntax before, in this example: https://scikit-image.org/docs/dev/auto_examples/applications/plot_3d_structure_tensor.html

I think it makes sense to display a proper title as opposed to a hyperlink within a sentence that should read smoothly... I admit I stayed between two stools in this case: I hesitated between "see tutorial: Title" and adapting the syntax to the example's title. Generally speaking, I'm in favour of using the imperative for example titles (just like commit messages), so there's a standard, plus it's engaging for the reader.